site/docs/v1/tech/core-concepts/tenants.adoc

```diff
@@ -208,6 +208,51 @@ The number of days before a child account that has not yet been verified by a pa
 +
 Required when the [field]#Delete unverified children# toggle is enabled.
 
+==== Multi-Factor
+
+image::tenant-configuration-mfa-authenticator-settings.png[Tenant Configuration - MFA authenticator settings,width=1200,role=shadowed top-cropped]
+
+===== Authenticator settings
+
+[.api]
+[field]#Enabled# [optional]#Optional#::
+When enabled, users may use an authenticator application to complete a two-factor authentication request.
+
+image::tenant-configuration-mfa-email-settings.png[Tenant Configuration - MFA email settings,width=1200,role=shadowed top-cropped]
+
+===== Email settings
+
+[.api]
+[field]#Enabled# [optional]#Optional#::
+When enabled, users may use an email address to complete a two-factor authentication request.
+
+[field]#Template# [required]#Required#::
+The email template to use when a two-factor authentication request is sent to the User's email address.
++
+Required when the [field]#Enabled# toggle is enabled.
+
+image::tenant-configuration-mfa-sms-settings.png[Tenant Configuration - MFA SMS settings,width=1200,role=shadowed top-cropped]
```

Comment:
I think shadowed is no longer needed.

Looks like this image is both top and bottom cropped so needs the bottom-cropped role as well.